cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.pb.go

```diff
@@ -16,17 +16,17 @@ limitations under the License.
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.19.1
+// 	protoc-gen-go v1.28.1
```

Comment:
Everything in this file is auto-generated using 
```
protoc \
  -I ./cluster-autoscaler \
  -I ./cluster-autoscaler/vendor \
  --go_out=. \
  --go-grpc_out=. \
  ./cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.proto
```
https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/externalgrpc#code-generation